### PR TITLE
Vending machines.

### PR DESCRIPTION
**Pull request by tortellinitony**
This PR does not change any sprites.

### CODE REVIEW

**Comment by Spacedong** [comment]
wat

**Review by j-mie — CHANGES_REQUESTED** [review]
Why are you deleting  hippiestation/sound/.gitkeep

File: hippiestation/code/game/machinery/vending.dm [hunk 1]
```diff
@@ -0,0 +1,506 @@
+#define STANDARD_CHARGE 1
```
Comment (j-mie) [inline hippiestation/code/game/machinery/vending.dm:1]: Do you need to mirror this entire file? How much of it did you change? Can you not just merge your changes and call procs from the tg vending.dm

**Review by j-mie — DISMISSED** [review]

File: hippiestation/code/game/machinery/vending.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/game/machinery/vending.dm:1]: its better this way

**Comment by ghost** [comment]
I'm missing the NanoUI changes, and you need to recompile nanoUI was well, rest looks good

**Comment by tortellinitony** [comment]
Oh yeah, forgot the template.

**Comment by tortellinitony** [comment]
I'll fix the map errors later.

**Review by ghost — APPROVED** [review]

**Pull request merged**